cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -160,6 +160,10 @@ spec:
           value: {{ .Values.csi.resizer.image | quote }}
 {{- end }}
 {{- end }}
+{{- if .Values.csi.podLabels }}
+        - name: ROOK_CSI_POD_LABELS
```

Comment:
We might want to consider separate labels for the rbd or cephfs drivers, but if there's not a demand for that separation, it's simpler to keep as a single setting.

Reply:
Yeah, I thought about that as well. I'll check on the "requirements" I have for this feature, though I think I might go for separate labels env vars for rbd and cephfs CSI components.